packages/core/src/Stores/notification-store.js

```diff
@@ -1293,6 +1300,21 @@ export default class NotificationStore extends BaseStore {
                 ),
                 type: 'warning',
             },
+            has_changed_two_fa: {
+                key: 'has_changed_two_fa',
+                header: localize('Logging out on other devices'),
+                message: has_enabled_two_fa ? (
+                    <Localize i18n_default_text="You've enabled 2FA on this device. You'll be logged out of your account on other devices (if any). Use your password and a 2FA code to log back in." />
+                ) : (
+                    <Localize i18n_default_text="You've disabled 2FA on this device. You'll be logged out of your account on other devices (if any). Use your password and a 2FA code to log back in." />
```

Comment:
you can conditionally display enable/disable to reduce code duplication.

Reply:
@matin-deriv changes made. 👍